### ISSUE TITLE
Bug: not possible to pass override default options

### ISSUE THREAD

**Issue by srghma** [JordanMartinez/purescript-node-execa#20]
I want Nothing to result in infinity

https://github.com/JordanMartinez/purescript-node-execa/blob/9d8b11cc3694e19d82c40f316f0b2dcd4fd3794f/src/Node/Library/Execa/GetStream.purs#L37

but if I will set Nothing - it will be overriden

https://github.com/JordanMartinez/purescript-node-execa/blob/9d8b11cc3694e19d82c40f316f0b2dcd4fd3794f/src/Node/Library/Execa.purs#L238

(this is me trying to finish https://github.com/srghma/purescript-tidy/pull/1/files#diff-09daa251f442ef54cabb8984167899cfceb4f53392275f554b5a06906eda48b5)

![2024-03-18-08am-43-07-screenshot](https://github.com/JordanMartinez/purescript-node-execa/assets/7573215/7c1dcf54-d4af-4064-b3cd-6d4f288e8ddd)

![2024-03-18-08am-41-50-screenshot](https://github.com/JordanMartinez/purescript-node-execa/assets/7573215/a1d94502-0151-4619-9848-3502311f931a)

I think options from 

https://github.com/JordanMartinez/purescript-node-execa/blob/9d8b11cc3694e19d82c40f316f0b2dcd4fd3794f/src/Node/Library/Execa.purs#L188-L198

should be moved to 

https://github.com/JordanMartinez/purescript-node-execa/blob/9d8b11cc3694e19d82c40f316f0b2dcd4fd3794f/src/Node/Library/Execa.purs#L149-L170

**Comment by JordanMartinez** [comment]
Would `options = Just Data.Number.infinity` work?

**Comment by srghma** [comment]
> Would `options = Just Data.Number.infinity` work?

yes, it works, this is how I fixed other issue in this repo (`stdout` was an empty string on `Normal 0`)
